.github/workflows/ios-pr.yml

```diff
@@ -1,10 +1,6 @@
 name: iOS Unit tests
 on:
   workflow_dispatch:
-  pull_request:
```

Comment:
I would remove the pipelines completely, as we won't use them.

Reply:
@igordmn , I made it to be aligned with other pipelines. They also aren't active and have `workflow_dispatch` launch trigger. The decision is up to you, so what we can do here:
- Merge current MR with no changes.
- Remove only `ios-pr.yml'.
- Remove all pipelines.

Reply:
Others are either "temporary" or came from AOSP.

let's remove `ios-pr.yml`

Reply:
Done